src/VisualStudio/Core/Def/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -134,6 +134,9 @@ private StreamingFindUsagesPresenter(
     public IClassificationFormatMap ClassificationFormatMap
         => _lazyClassificationFormatMap.Value;
 
+    private static bool IsPrimary(DefinitionItem definition)
+        => definition.Properties.ContainsKey(DefinitionItem.Primary);
```

Comment:
we already had this data in the definitionItem.  We were using it for other featuers.  but hadn't updated FAR properly.